modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/components/notification-sidebar/index.tsx

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable no-console */
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import classNames from 'classnames';
+import {useEffect, useState} from 'react';
+
+import './index.scss';
+import {getApplicationsById} from '../../services/Application';
+import {
+	getUserNotification,
+	putUserNotificationRead,
+} from '../../services/notification';
+import {getLiferaySiteName} from '../../utils/liferay';
+import {PostType} from './postTypes';
+
+const initialPagination = {order: 'desc', page: 1, pageSize: 2, totalCount: 0};
+
+const NotificationSidebar: React.FC = () => {
+	const [posts, setPosts] = useState<PostType[]>([]);
+	const [totalCount, setTotalCount] = useState<number>(
+		initialPagination.totalCount
+	);
+	const [postsWithLinks, setArrayOfLinks] = useState<PostType[]>([]);
+	const [page, setPage] = useState<number>(initialPagination.page);
+	const hasMorePostsToLoad = posts.length < totalCount;
+
+	const notificationCategory = {
+		Application: 'Application ',
+	};
+
+	const creatRoute = (externalReferenceCode: string, entity: string) => {
+		const currentSiteName = getLiferaySiteName();
+		const link = `${currentSiteName}/${entity}?externalReferenceCode=${externalReferenceCode}`;
+
+		return link;
+	};
```

Comment:
Could you add this to a component file? 
```suggestion
	const createUrl = (externalReferenceCode: string, entity: string) => {
		const currentSiteName = getLiferaySiteName();
		const link = `${currentSiteName}/${entity}?externalReferenceCode=${externalReferenceCode}`;

		return link;
	};
```

Reply:
Done, thx!